### ISSUE TITLE
show a label in search results for resources without a BF or RDA title property

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#2869]
Sinopia is optimized for BF (and RDA) so when doing a search, we will display a resource's "title" label in the first column of search results as long as the resource has one of the BF or RDA properties explained in https://github.com/LD4P/sinopia/wiki/Searching-in-Sinopia

There are more users interested in Sinopia for describing resources using properties from ontologies other than BF and RDA. When those resources are created and searched, the search results under "Label / ID" shows the Sinopia URI for the resource, which does not offer much information to the user about what the result is.

Example (search for * on stage, limit to institution "Other", sort by date, look at some records created july 19, 2021):
<img width="1229" alt="Screen Shot 2021-07-21 at 8 38 23 AM" src="https://user-images.githubusercontent.com/23069165/126518976-ae1c5d60-b12c-4389-884d-796db72ddaa9.png">

can we provide a label instead of / in addition to the resource URI?

i know there is no property across all possible ontologies that is going to provide the label.

possible solutions:
(1) use "first" label encountered in RDF?
(2) look thru RDF for any property or field label with string "title" and use that, since many ontologies do have a 'title'

**Comment by justinlittman** [comment]
RDF does not have a notion of "first".

Any "magic" is just going to precipitate user confusion / questions. Can the title be specified for other ontologies? Perhaps this should be made a configuration (e.g., an LDPath)?

**Comment by michelleif** [comment]
Yes I like the idea of a configuration, even a user-editable one, like we have for the character map.

**Comment by michelleif** [comment]
since this applies to Sinopia lookups (as well as searches) we need to work on this in WC4 because it is part of supporting creating a Person entity in Sinopia then looking it up in a Work resource in order to use the URI (because we are taking away ability to put literal in URI field which was previous practice of how to handle people not found in external authority sources)

if we don't want to implement the user-configurable way of adding properties whose objects can be treated as labels, we could instead just collect the most common ones and specify them

**Comment by michelleif** [comment]
see #2933 as potential solution for "what label to display?"

**Comment by michelleif** [comment]
will be handled by #2933 so closing
